src/Kunstmaan/TranslatorBundle/Resources/views/Translator/inline_edit.html.twig

```diff
@@ -1,6 +1,8 @@
-{% if adminlist.configurator.canEditInline(row) %}
+{% spaceless %}
```

Comment:
The `spaceless` tag is deprecated, use `{% apply spaceless%}{% endapply%}` instead